docs/source/worker.rst

```diff
@@ -184,6 +185,9 @@ file:
          pause: 0.80  # fraction at which we pause worker threads
          terminate: 0.95  # fraction at which we terminate the worker
 
+It is possible to individually disable any of these by setting its value to False.
+Setting 'target' while leaving 'spill' active disables the spill hysteresis cycle.
+
```

Comment:
I'm not entirely sure if this is supposed to be possible. Why would we want to disable target and keep spilling? I don't think our documentation is currently sufficient to let users decide when to turn off the one or the other. Documenting this in code is the one thing but this is likely the go-to top-level documentation for spill-to-disk and I'm concerned that this paragraph will just confuse people more.

Reply:
This makes sense; I'll remove the paragraph